scripts/deploy

```diff
@@ -94,6 +97,14 @@ else
     echo "Failed to install python virtual environments, you may need to install this manually"
     exit 1;
   fi
+  if ! which npm; then
+    echo "npm needs to be installed to use this script"
+    exit 1;
+  fi
+  if [ $(npm version | cut -f1 -d.) -lt 9 ]; then
+    echo "npm needs to be version 9 or above to use this script"
```

Comment:
truly unclear what version of `npm` works as a minimum, but it definitely works on 9 and definitely breaks on 5. not terribly interested in spending a bunch of time to find out the minimum either... @linknum23 what's on your localhost? lol

Reply:
v9.11.2

Reply:
With 0.11.2: `npm does not support Node.js v9.11.2`

Reply:
we established @linknum23 actually runs npmv major version 6, _and_ that he does not want his machine's built-in to be targeted because he uses `nvm`. I think it's sane enough to just ship with `npm` >= 9.

Reply:
I think checking and failing if npm version is too old makes more sense for npm. Trying to install it from apt is complicated on some machines that aren't mine.

Reply:
yeah, it's a mess. alright, I moved it to its own conditional check in the latest revision.